content/zh/guide/v11/getting-started.md

```diff
@@ -0,0 +1,275 @@
+---
+title: 入门指南
+description: 如何开始使用 Preact。我们将学习如何设置工具（如果需要）并开始编写应用程序
+---
```

Comment:
You can pretty much copy all of `/content/zh/guide/v10` to `/v11`, that's pretty much what we've done with the English. _I think_ the only file that deviates is the 'Upgrade Guide' page?

Reply:
Off the top of my head, I believe `/upgrade-guide.md` is the only one that'll need to be completely rewritten, with `/api-reference.md` removing a tiny bit of info and `/typescript.md` changing a little too. You can take a look at #1298 and skim through the commits for a better idea.

I think the rest were copied without any changes, but it's not a big deal if we find some differences later. Just trying to save you some time is all.